spec/javascripts/admin/modules/edition-form.spec.js

```diff
@@ -283,15 +283,15 @@ describe('GOVUK.Modules.EditionForm', function () {
         editionForm.init()
       })
 
-      it('shows the speech delivered_on in future warning to the user when they select a date in the future', function () {
+      it('shows the speech delivered_on in future warning to the user when they input a date in the future', function () {
         const deliveredOnFieldset = form.querySelector('#edition_delivered_on')
-        deliveredOnFieldset.querySelector('select').value = '1'
-        deliveredOnFieldset.querySelectorAll('select')[1].value = '1'
-        deliveredOnFieldset.querySelectorAll('select')[2].value =
-          currentYear + 1
+        console.log(deliveredOnFieldset.innerHTML)
```

Comment:
```suggestion
```

Reply:
Arrgh, there's definitely a linter rule we can add to eslint for this...